content/messaging/agent-chat-api/changelog/index.mdx

```diff
@@ -11,64 +11,86 @@ This document is the record of all the changes in the **Agent Chat API**, Web an
 
 ## [v3.2] - 2020-06-18
 
-### Added
+### Chats
 
-- There's a new method **List Agents For Transfer** ([Web](/messaging/agent-chat-api/v3.2/#list-agents-for-transfer) & [RTM](/messaging/agent-chat-api/v3.2/rtm-reference/#list-agents-for-transfer)).
-- There's a new method **Get Customer** ([Web](/messaging/agent-chat-api/v3.2/#get-customer) & [RTM](/messaging/agent-chat-api/v3.2/rtm-reference/#get-customer)) that returns info about a given Customer.
-- There's a new push [`chat_unfollowed`](/messaging/agent-chat-api/v3.2/rtm-reference/#chat_unfollowed) complementary to the **Unfollow Chat** method.
-- There's a new `include_chats_without_threads` filter in the  **List Chats** ([Web](/messaging/agent-chat-api/v3.2/#list-chats) & [RTM](/messaging/agent-chat-api/v3.2/rtm-reference/#list-chats)) method.
+- The **Get Archives** method was renamed to **List Archives**.
 - There's a new `agents` filter in the **List Archives** ([Web](/messaging/agent-chat-api/v3.2/#list-archives) & [RTM](/messaging/agent-chat-api/v3.2/rtm-reference/#list-archives)) method. It replaces the `agents_ids` filter.
 - There's a new `events.types` filter in the **List Archives** ([Web](/messaging/agent-chat-api/v3.2/#list-archives) & [RTM](/messaging/agent-chat-api/v3.2/rtm-reference/#list-archives)) method.
 - The **List Archives** ([Web](/messaging/agent-chat-api/v3.2/#list-archives) & [RTM](/messaging/agent-chat-api/v3.2/rtm-reference/#list-archives)) method has new pagination with hashed page IDs replacing the old pagination format. Pagination is now uniform across the whole Agent Chat API.
-- There's a new method **List Threads** ([Web](/messaging/agent-chat-api/v3.2/#list-threads) & [RTM](/messaging/agent-chat-api/v3.2/rtm-reference/#list-threads)).
 - There's a new `highlights` parameter in the **List Archives** ([Web](/messaging/agent-chat-api/v3.2/#list-archives) & [RTM](/messaging/agent-chat-api/v3.2/rtm-reference/#list-archives)) method. It allows you to highlight the match of `filters.query` with an HTML tag.
-- There's a new push [`queue_positions_updated`](/messaging/agent-chat-api/v3.2/rtm-reference/#queue_positions_updated) with updated positions and wait times for queued chats.
 
-### Changed
+- The **Get Chats Summary** method was renamed to **List Chats**.
+- There's a new `include_chats_without_threads` filter in the  **List Chats** ([Web](/messaging/agent-chat-api/v3.2/#list-chats) & [RTM](/messaging/agent-chat-api/v3.2/rtm-reference/#list-chats)) method.
+- The `order` field was renamed to `sort_order` in **List Chats**.
+- The `order` field in the `chat` object was removed from the responses of the **List Chats** method.
+
```

Comment:
I have these empty lines to help me group changes, but I will remove them in the final version.